### ISSUE TITLE
Dedicated Docs page outdated

### ISSUE THREAD

**Issue by SimonFischer04** [DrozmotiX/ioBroker.esphome#202]
The docs here https://drozmotix.github.io/languages/en/Adapter/ESPHome/ are pretty outdated with the recent changes.

How should we proceed here? Of course, we could "simply update it", but as a lot of the info is already in the README, wouldn't that be kind of duplicated and annoying to update 2 sources every time something changes?

**Comment by DutchmanNL** [comment]
> The docs here https://drozmotix.github.io/languages/en/Adapter/ESPHome/ are pretty outdated with the recent changes.
> 
> How should we proceed here? Of course, we could "simply update it", but as a lot of the info is already in the README, wouldn't that be kind of duplicated and annoying to update 2 sources every time something changes?

honestly... struggling with a decision here .... im a still not 100% convinced if the separate docu page has an added value (as it has much more options than the GitHub readme etc) or to keep all on git....

info should only be on one space, that's a fact and not duplicated or different. Question is, which source/layout etc is the best

**Comment by SimonFischer04** [comment]
> honestly... struggling with a decision here .... im a still not 100% convinced if the separate docu page has an added value (as it has much more options than the GitHub readme, like language/navigation etc) or to keep all on git....

Yeah. Same struggle here - not sure whats the best way either, that was why I created the issue. Dedicated docs page has more features, but Readme is whats shown in the adapter info also...

> info should only be on one space, that's a fact and not duplicated or different. Question is, which source/layout etc is the best

I 100% agree
